python/test/sensor_TEST.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# Copyright (C) 2023 Open Source Robotics Foundation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from gz.common import set_verbosity
+from gz_test_deps.sim import (K_NULL_ENTITY, TestFixture,
+                              Joint, Model, Sensor, World, world_entity)
+from gz_test_deps.math import Pose3d
+
+
+class TestSensor(unittest.TestCase):
+    post_iterations = 0
+    iterations = 0
+    pre_iterations = 0
+
+    def test_model(self):
+        set_verbosity(4)
+
+        file_path = os.path.dirname(os.path.realpath(__file__))
+        fixture = TestFixture(os.path.join(file_path, 'joint_test.sdf'))
+
+        def on_post_udpate_cb(_info, _ecm):
+            self.post_iterations += 1
+
+        def on_pre_udpate_cb(_info, _ecm):
+            self.pre_iterations += 1
+            world_e = world_entity(_ecm)
+            self.assertNotEqual(K_NULL_ENTITY, world_e)
+            w = World(world_e)
+            m = Model(w.model_by_name(_ecm, 'model_test'))
+            j = Joint(m.joint_by_name(_ecm, 'joint_test'))
+            sensor = Sensor(j.sensor_by_name(_ecm, 'sensor_test'))
+            # Entity Test
+            self.assertNotEqual(K_NULL_ENTITY, sensor.entity())
+            # Valid Test
+            self.assertTrue(sensor.valid(_ecm))
+            # Name Test
+            self.assertEqual('sensor_test', sensor.name(_ecm))
+            # Pose Test
+            self.assertEqual(Pose3d(0, 1, 0, 0, 0, 0), sensor.pose(_ecm))
+            # Topic Test
+            # This is None because the entity does not have a
+            # components::SensorTopic component.
```

Comment:
Do we know why this is? If it's because this is a Joint sensor, can you create an issue so we can fix it later?

Reply:
After a little lookup, I'm not completely sure why this is happening, it could be due to the reason you just commented but I'm not 100% sure of it. How should we proceed on this case?

Reply:
Looks like they're created by sensor systems (https://github.com/gazebosim/gz-sim/blob/87e87bd6676538bb963ccd793c89a1b25cffa270/src/systems/force_torque/ForceTorque.cc#L149), so we'd need the `ForceTorque` sensor in this case. 

Putting
```
    <plugin filename="gz-sim-physics-system" name="gz::sim::systems::Physics" />
    <plugin filename="gz-sim-forcetorque-system" name="gz::sim::systems::ForceTorque" />
```
to the .sdf file should fix this.

Reply:
Ok, I'll try it out